lib/active_model_serializers.rb

```diff
@@ -51,6 +51,7 @@ def silence_warnings
 require 'active_model/serializer'
 require 'active_model/serializable_resource'
 require 'active_model/serializer/version'
+require 'active_model_serializers/test/serializer'
```

Comment:
IMHO, this should only be loaded in the test environment.  Sell me on always loading it, and why it won't cause issues in production (including the railtie `ActionController::TestCase.send(:include, ActiveModelSerializers::Test::Serializer`)

- ref to [previous comment lost in the diffs](https://github.com/rails-api/active_model_serializers/pull/1099#discussion_r42064156)